src/test/java/bowling/ui/OutputCuiTest.java

```diff
@@ -0,0 +1,62 @@
+package bowling.ui;
+
+import bowling.domain.Game;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class OutputCuiTest {
+
+    private Game game;
+
+    @BeforeEach
+    void setUp() {
+        game = new Game("tes");
+        game.throwBall(10); // 1
+        game.throwBall(10);
+        game.throwBall(10);
+        game.throwBall(10);
+        game.throwBall(10); // 5
+        game.throwBall(10);
+        game.throwBall(10);
+        game.throwBall(10);
+        game.throwBall(10); // 9
```

Comment:
사소해보일 수 있지만, 이러한 코드의 중복도 줄여보는 건 어떨까요?
```suggestion
        for (int frame = 0; frame < 9; frame++) {
            game.throwBall(10);
        }
```